civihr_employee_portal/features/civihr_default_permissions/civihr_default_permissions.features.user_permission.inc

```diff
@@ -23,11 +23,11 @@ function civihr_default_permissions_user_default_permissions() {
   $permissions['access AJAX API'] = array(
     'name' => 'access AJAX API',
     'roles' => array(
```

Comment:
Just to confirm, you updated this by downloading the feature from the Drupal admin and then extracting the files, right?